crates/zoon/src/element/button.rs

```diff
@@ -8,6 +8,60 @@ use std::marker::PhantomData;
 
 make_flags!(Label, OnPress);
 
+/// Html Button for your web page.
+/// You can create a new button by using its constructor and chain styling.
+///
+/// # Examples
+/// ```no_run
+/// use zoon::*;
+/// let button = Button::new().s(Align::center()).s(Padding::all(5));
+/// ```
+/// You can also create your button with specific events as well and update the styling the way you need by using [signals](https://crates.io/crates/futures-signals)
+///
+/// 1 - Create signal for local state management.
+///
+/// 2 - The variable **`hovered`** gets updated when the user hovers the button
+///
+/// 3 - The variable **`hovered_signal`** is the notification send by signal
+/// that will actually determine the background color for the button.
+///
+/// # Example
+/// ```no_run
+/// use zoon::{named_color::*, *};
+/// let (hovered, hovered_signal) = Mutable::new_and_signal(false);
+///
+/// let button = Button::new().s(Background::new()
+///     .on_hovered_change(move |is_hovered| hovered.set_neq(is_hovered))
+///     .color_signal(hovered_signal.map_bool_signal(|| GREEN_7, || GREEN_8)));
+/// ```
+/// It is also possible to with local state and make quick updates.
+///
+/// # Example
+/// ```no_run
+/// use zoon::{*, named_color::*};
+/// use std::{rc::Rc,};
+/// use std::borrow::Cow;
+///     
+/// let click_count = Mutable::new(0);
+///
+/// // Create a title that gets updated when counting changes.
+/// let title = click_count.signal().map(|count| {
+///         if count == 0 {
+///             return Cow::from("Click me!");
+///         }
+///         Cow::from(format!("Clicked {}x", count))
+///     });
+///
+/// // Create a raw that contains the button with the label changing and the press method that increment.
```

Comment:
Typo: `raw` => `row`.
What about "Create a row with a button. The button's `label` is _dynamic_ (changing according to the `title` signal). Click the button to increment the `click_count` value." => Maybe too long but the user learns about dynamic vs static properties and it's a bit more expressive. I'm not sure if it's better or worse. Your choice.